parser/src/lexer.rs

```diff
@@ -20,8 +20,8 @@ pub enum LexicalError {
 
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct Location {
-    row: usize,
-    column: usize,
+    pub row: usize,
```

Comment:
It looks like we only need these to be readable; would creating accessors be preferable?

Reply:
Valid point!